IMS/IMS/Controllers/InventoryController.cs

```diff
@@ -13,11 +13,62 @@ public InventoryController(IInventoryRepository InventoryRepository)
         {
             _InventoryRepository = InventoryRepository;
         }
+
         [HttpGet]
-        public IActionResult GetAll()
+        public async Task<IActionResult> Get(int page = 1, int limit = 10)
+        {
+            var inventory = await _InventoryRepository.GetInventoryAsync(page,limit);
+            return Ok(inventory);
+        }
+
+        [HttpGet("{inventory_id}")]
+        public async Task<IActionResult> Get(int inventoryId)
+        {
+            var inventory = await _InventoryRepository.GetInventoryByIdAsync(inventoryId);
+            return Ok(inventory);
+        }
+
+        [HttpPost]
+        public async Task<IActionResult> Post(Inventory inventory)
+        {
+            await _InventoryRepository.CreateInventoryAsync(inventory);
+
+            return Ok(inventory);
+        }
+
+        [HttpPut("{inventory_id}")]
+        public async Task<IActionResult> Put(int inventoryId, Inventory inventoryDTO)
```

Comment:
Update all dto parameters to match .NET naming standards.
```suggestion
        public async Task<IActionResult> Put(int inventoryId, Inventory inventoryDto)
```